sycl/test/devicelib/math_utils.hpp

```diff
@@ -1,25 +1,33 @@
 #ifndef MATH_UTILS
 #include <complex>
 #include <limits>
-using namespace std;
 // T must be float-point type
 template <typename T>
 bool is_about_FP(T x, T y) {
-  if (x == y)
-    return true;
+  bool ret;
+
+  // At least one input is nan
+  if (std::isnan(x) || std::isnan(y))
+    ret = std::isnan(x) && std::isnan(y);
+
+  // At least one input is inf
+  else if (std::isinf(x) || std::isinf(y))
```

Comment:
I found the code with early exit more readable than this version.
```suggestion
  // At least one input is nan
  if (std::isnan(x) || std::isnan(y))
    return std::isnan(x) && std::isnan(y);

  // At least one input is inf
  if (std::isinf(x) || std::isinf(y))
```

Reply:
Fix for fixing this, but could you apply this comment to the file, please?

Reply:
Hi, Alexey.
I am sorry to make things complicated. I am new user.
Thanks for your patience.